trimer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+# Define the TrimmerDecorator class that inherits from Decorator
+require_relative 'base_decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    name = @nameable.correct_name
+    name.lenght > 10 ? name.slice(0, 10) : name
```

Comment:
```suggestion
    name.length > 10 ? name.slice(0,9) : name
```


- [ ] Kindly update the word `length` from lengt to length in your method above 👍🏿 
- [ ] Based on the requirements, the range in your slice method should be `0,9` please update it 🤗